code/__DEFINES/recipes.dm

```diff
@@ -0,0 +1,4 @@
+/// If the recipe deletes contents of the crafted item (Only works with storage items, used for ammo boxes, donut boxes, internals boxes, etc)
+#define RECIPE_DELETE_CONTENTS (1<<0)
```

Comment:
By default, all recipes delete the contents of the resulting object, so this should be a flag for not deleting the contents, which only affects a handful and is easier to maintain.

Reply:
Makes sense